kubetest/main.go

```diff
@@ -717,6 +717,9 @@ func prepareGcp(o *options) error {
 		os.Setenv("NODE_OS_DISTRIBUTION", nod)
 	}
 	if o.gcpProject == "" {
+		log.Print(`--gcp-project is missing, try to fetch a project from boskos.
+				  (for local run please point --gcp-project to your dev project)`)
```

Comment:
"trying to fetch", and "for local runs set --gcp-project to your dev project"

Reply:
done